### PR TITLE
Implemented a button test

### PR DESCRIPTION
**Pull request by taminob**

### CODE REVIEW

File: packages/MorphicAPIExplorer-Tests.package/MEXOverViewTestsWithFullCategories.class/instance/setUp.st [hunk 1]
```diff
@@ -6,5 +6,5 @@ setUp
 	maxTotalRows := 20.
 	full  := OrderedCollection new.
 	defaultCollectionSize := 20.
-	1 to: defaultCollectionSize do: [:anIndex | full add: MEXCategory new].
+	1 to: defaultCollectionSize do: [:anIndex | full add: (MEXCategory newNamed: 'a' havingImage: MEXScalableImageMorph new havingDefaultMorph: Morph new withMethodList: {MEXMethod newWithName: 'b'})].
```
Comment (Zyranix) [inline packages/MorphicAPIExplorer-Tests.package/MEXOverViewTestsWithFullCategories.class/instance/setUp.st:9]: Very long line

File: packages/MorphicAPIExplorer-Tests.package/MEXOverViewTestsWithFullCategories.class/instance/testCategoryButtonToExplorationView.st [hunk 2]
```diff
@@ -0,0 +1,11 @@
+tests
+testCategoryButtonToExplorationView
+
+	| view openExplorationViews openOverViews |
+	view := builder build: overView.
+	view openInWorld.
+	openExplorationViews := (self getOpenWindowsWithModel: MEXToolExplorationView) size.
+	openOverViews := (self getOpenWindowsWithModel: MEXToolOverView) size.
+	((view submorphs select: [:each | each isKindOf: MEXPluggableButtonMorph]) select: [:each | each label isKindOf: MEXScalableImageMorph]) first performAction.
```
Comment (Zyranix) [inline packages/MorphicAPIExplorer-Tests.package/MEXOverViewTestsWithFullCategories.class/instance/testCategoryButtonToExplorationView.st:9]: dont use each, but a|anWORD (-> coding standard)

File: packages/MorphicAPIExplorer-Tests.package/MEXTestCaseForUI.class/instance/getOpenWindowsWithModel..st [hunk 3]
```diff
@@ -0,0 +1,4 @@
+tests
+getOpenWindowsWithModel: aClass
+
+	^ ActiveWorld submorphs  select: [:each|( each isKindOf: PluggableSystemWindow) and: (each model isKindOf: aClass)]
```
Comment (Zyranix) [inline packages/MorphicAPIExplorer-Tests.package/MEXTestCaseForUI.class/instance/getOpenWindowsWithModel..st]: dont use each, but a|anWORD (-> coding standard)

File: packages/MorphicAPIExplorer-Tests.package/MEXOverViewTestsWithFullCategories.class/instance/testCategoryButtonToExplorationView.st (hunk 2 above)
Comment (Zyranix) [inline packages/MorphicAPIExplorer-Tests.package/MEXOverViewTestsWithFullCategories.class/instance/testCategoryButtonToExplorationView.st]: don't use each

File: packages/MorphicAPIExplorer-Tests.package/MEXOverViewTestsWithFullCategories.class/instance/setUp.st (hunk 1 above)
Comment (Zyranix) [inline packages/MorphicAPIExplorer-Tests.package/MEXOverViewTestsWithFullCategories.class/instance/setUp.st]: very long line

**Review by Zyranix — CHANGES_REQUESTED** [review]
test looks good, only small refactoring stuff should be fixed

File: packages/MorphicAPIExplorer-Tests.package/MEXOverViewTestsWithFullCategories.class/instance/setUp.st [hunk 4]
```diff
@@ -6,5 +6,5 @@ setUp
 	maxTotalRows := 20.
 	full  := OrderedCollection new.
```
Comment (Zyranix) [inline packages/MorphicAPIExplorer-Tests.package/MEXOverViewTestsWithFullCategories.class/instance/setUp.st:7]: test

**Review by Zyranix — APPROVED** [review]
Looks fine now

**Review by sinahenning — COMMENTED** [review]
Top!

**Review by sinahenning — APPROVED** [review]

**Pull request merged**